requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==4.1.2
+pre-commit==2.20.0

```

Comment:
Add these two to master directly. This PR should have only markdown requirement added